compiler/test/compilable/test22727.c

```diff
@@ -1,11 +1,18 @@
 // https://issues.dlang.org/show_bug.cgi?id=22727
 
+
 int fooc(int a) { return a; }
 
+#ifdef _WIN32
```

Comment:
How come this worked before adding these guards?

Reply:
I am surprised that the guard didn't exist in the first place, too.

Reply:
I think I get it now. It seems that __stdcall is recognized by the compiler irrespective of the platform where you are compiling on, whereas you are adding the alias only for version windows. For more info, check: https://github.com/dlang/dmd/pull/13614

Reply:
cc @WalterBright - is there any reason to hava stdcall being recognized on non-windows platforms?

Reply:
@RazvanN7 if you request I remove the guard. Without it CI test fail for non-linux systems.

Reply:
@aferust It seems that @WalterBright has implemented stdcall so that it is recognized by the parser irrespective of the target platform. Now I wonder if this was simply an overlook or if it was intended behavior. Depending on the answer to that question we will know if this PR needs to be implemented in a way that _stdcall is recognized irrespective of the platform or if it should be windows only.

I guess if we want to be consistent, the _stdcall alias should defined for all platforms, but I want confirmation from @WalterBright before doing that.

Reply:
stdcall is just one of the few ABIs possible with on x86-32.

https://godbolt.org/z/osxa3qhcs

It's not rejected, but rather a warning on x86-64 and other targets.  ImportC follows the convention that if it's only warned about, then just accept and ignore.